IsCourseActivitySyncEnabled for LearningProvider (#21464)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/learningprovider.md

|longLogoWebUrlForLightTheme|String|The long logo URL for the light mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering  within the Viva Learning app. Required.|
|squareLogoWebUrlForDarkTheme|String|The square logo URL for the dark mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering within the Viva Learning app. Required.|
|squareLogoWebUrlForLightTheme|String|The square logo URL for the light mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering within the Viva Learning app. Required.|

> |isCourseActivitySyncEnabled|Boolean|Indicates whether the course activities will be ingested for this provider or not. Default value is `false`. Course activities for a provider cannot be ingested if this value is false. This value needs to be updated to true when the provider wants to ingest courseactivities.|

Reviewer: Perhaps, we can have a better word choice here. I am not clear, on who could be a provider. Please share an example.

Reviewer: Please add the changelog.

Reviewer 2: Added changelog. This is a new property added to existing learning provider api.

> |isCourseActivitySyncEnabled|Boolean|Indicates whether learning course activity records will be ingested for this provider or not. Default value is `false`. Learning course activities for a provider cannot be ingested if this value is false. This value needs to be updated to true when the provider wants to ingest learningCourseActivities.|

Reviewer: ```suggestion
|isCourseActivitySyncEnabled|Boolean|Indicates whether a provider can ingest learning course activity records. The default value is `false`. Turn to true to make learningCourseActivities for this provider. .|
```
